### PR TITLE
Laboratory work #8, Budanova Tatiana - 20FPL1

### PR DESCRIPTION
**Pull request by tanynova99**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+--extra-index-url https://download.pytorch.org/whl/cpu
```
Comment (SalnikovIgor) [inline requirements.txt:1]: Surprisingly, the tests are green

**Review by SalnikovIgor — CHANGES_REQUESTED** [review]

File: lab_7_llm/start.py [hunk 2]
```diff
@@ -15,4 +15,4 @@ def main() -> None:
 
 
 if __name__ == "__main__":
-    main()
+    main()
```
Comment (SalnikovIgor) [inline lab_7_llm/start.py]: Why are these changes from lab 7 still here?

File: lab_8_llm/main.py [hunk 3]
```diff
@@ -34,6 +25,7 @@ class RawDataImporter(AbstractRawDataImporter):
     """
     A class that imports the HuggingFace dataset.
     """
+    _raw_data: DataFrame
```
Comment (SalnikovIgor) [inline lab_8_llm/main.py]: Are you sure this announcement is necessary?

File: lab_8_llm/main.py [hunk 4]
```diff
@@ -43,6 +35,18 @@ def obtain(self) -> None:
         Raises:
             TypeError: In case of downloaded dataset is not pd.DataFrame
         """
+        self._raw_data = load_dataset(self._hf_name,
+                                      split='train').to_pandas()
+
+    @property
+    def raw_data(self) -> DataFrame:
```
Comment (SalnikovIgor) [inline lab_8_llm/main.py:47]: And this?

**Review by SalnikovIgor — CHANGES_REQUESTED** [review]

File: lab_7_llm/start.py (hunk 2 above)
Comment (tanynova99) [reply lab_7_llm/start.py]: I reverted all of them

File: lab_8_llm/main.py (hunk 3 above)
Comment (tanynova99) [reply lab_8_llm/main.py]: They were in the last project, I will try without them

File: lab_8_llm/main.py (hunk 4 above)
Comment (tanynova99) [reply lab_8_llm/main.py:47]: like previous project lab_7, without it mypy tests crash

File: lab_8_llm/main.py (hunk 3 above)
Comment (tanynova99) [reply lab_8_llm/main.py]: update: like previous project lab_7, without it mypy tests crash

Comment (SalnikovIgor) [reply lab_8_llm/main.py]: Need to add a check: `_raw_data` containt `None`? If yes, then rise error

File: lab_8_llm/main.py (hunk 4 above)
Comment (SalnikovIgor) [reply lab_8_llm/main.py:47]: See the comment above

**Review by SalnikovIgor — CHANGES_REQUESTED** [review]
Fix tests

File: lab_8_llm/main.py (hunk 3 above)
Comment (tanynova99) [reply lab_8_llm/main.py]: okay

File: lab_8_llm/main.py (hunk 4 above)
Comment (tanynova99) [reply lab_8_llm/main.py:47]: did it!

**Pull request closed without merging**